homeassistant/components/statsd.py

```diff
@@ -57,8 +57,16 @@ def statsd_event_listener(event):
         except ValueError:
             return
 
+        states = dict(state.attributes)
+
         _LOGGER.debug('Sending %s.%s', state.entity_id, _state)
-        statsd_client.gauge(state.entity_id, _state, sample_rate)
+        statsd_client.gauge("%s.state" % state.entity_id, _state, sample_rate)
```

Comment:
In order to avoid splitting the stat tree earlier, making it hard to use graphite to group up the stats in a single tree I may just add a log_attributes: true config to the statsd configuration. If it's set it will change the stat to .state and .attribute_name otherwise it will be the current setup